sentinel.md

```diff
@@ -7,4 +7,85 @@ This system contract performs two important tasks:
 The process of metering is [explained in its own chapter](./metering.md).
 
 The Sentinel is a system contract which means its code is fixed and is part of the genesis or hardfork ewasm is enabled on.
-As a result, changing or updating it requires a hard fork.
+As a result, changing or updating it requires a hard fork. In order to facilitate a more dynamic ecosystem a proposal follows
+here to introduce upgradability for the Sentinel contract.
+
+## Governance (upgradability)
+
+The Sentinel contract is split into two parts:
```

Comment:
Might be worth mentioning the `SentinelGovernance` contract/interface here as well. It's a little confusing to say there are two parts, then introduce a third part further down.